File: Nested for loop Basic-2/code2.c

#include <stdio.h>

void main (){

	int row ;
	printf("enter no of row: ");
	scanf("%d",&row);

	for(int i =0; i < row ; i++){

		for(int a = 0;a < row ; a++){

			printf("\t%d\t",row - a);

			
		}
		printf("\n\n");
	}


}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/10/code2.java
import java.util.Scanner;

class TriangleDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		for(int i = 0; i < row ; i++){

			for(int j = 0 ; j <= i ; j++ ){

				System.out.print((j+1)+"		");
			}

				System.out.print("\n\n");

		}

	}

}
